sqe-database-access/Queries/Image.cs

```diff
@@ -121,6 +147,19 @@ internal class Result
         }
     }
 
+    internal static class InstitutionImagesQuery
+    {
+        public const string GetQuery = @"
+SELECT image_catalog.object_id AS Name, 
+       CONCAT(image_urls.url, SQE_image.filename, '/full/150,/0/', image_urls.suffix) AS Thumbnail,
```

Comment:
Hmmm, do you really want to keep the IIIF protocol implementation in the SQL queries? I'm actually not sure whether we want to keep it in the backend *at all*. Why not let the API clients decide which image size they want?

Reply:
I have no problems with changing this.  I also don't love building this thumbnail address server side.  I'll give it some thought, maybe we should just give the bare IIIF URL address (without the sizing stuff).